core/src/main/java/org/opensearch/sql/planner/ExternalizablePlan.java

```diff
@@ -23,11 +23,11 @@
  *   </li>
  *   <li>
  *     Overwrite {@link #getPlanForSerialization} to return
- *     another instance of {@link SerializablePlan}.
+ *     another instance of {@link ExternalizablePlan}.
```

Comment:
Why rename?
I feel `SerializablePlan` is self-describing, while `ExternalizablePlan` is not.

Reply:
I'm of two opinions here -- `ExternalizablePlan` is less obvious but more consistent with Java serialization API name convention. `SerializablePaln` is more obvious but implies that the plan implements `Serializable` which is not the case.